### ISSUE TITLE
Template is erasable 

### ISSUE THREAD

**Issue by hugochien** [mobiscribe/mobiscribers#4]
Users reported that when they go to erase something on my 'page', the line underneath it is also erased

Version: 5.25.3827

**Comment by ektien** [comment]
Yes I saw this today. It happened to me when I was using a picture as template. Also the notefile contains more than 20 pages of drawing.

I created a new note file and it went back to normal.
